starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+body {
+  font-family: Helvetica, Arial, Sans-Serif;
+}
+
+.header {
+  height: 350px;
+  background-image: url('../images/apple-pie.jpg');
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 100px 0px 0px 0px;
+}
+
+.bordure {
+  border-bottom:1px solid black;
+  vertical-align: center;
+  padding: 25px 0px 0px 0px;
+}
+
+.header h1 {
+  font-size: 50px;
+  font-family: arial;
+  Color: white;
+  text-align: center;
+}
+
+h2{
+  font-size: 20px;
+  font-family: helvetica;
+  font-style:normal;
+  vertical-align: center;
+  padding : 20px 0px 0px 0px;
+}
+
+.paramother{
+  margin: auto;
+  vertical-align: center;
+}
+
+.paraMother pre {
```

Comment:
classes are case sensitive though, so `paramother` and `paraMother` are two different classes:

```html
<p class="paramother paraMother"></p>
```

It's better to do all lower case classes for the moment (kebab case)